docs/basics.md

```diff
@@ -460,7 +460,7 @@ This is an **essential feature to debug flaky tests**, as you can analyze them i
 > ℹ To enable pause after a test without a plugin use `After(pause)` inside a test file.
 
 
-### Screenshot on Failure
+### Screenshot on Failure <Badge text="Enabled by default"/>
```

Comment:
Do we need to add this badge to all enabled by default plugins?

What about disabled?

Reply:
Removed the badge